luda/editor/client/src/timeline/DefaultTrackBody.ts

```diff
@@ -32,22 +49,134 @@ export const DefaultTrackBody: Render<
       onMouseUp(event) {
         if (event.button === MouseButton.right) {
           const clickMs =
-            state.layout.startMs + event.translated.x * state.layout.msPerPixel;
-          state.contextMenu = {
+            state.timelineState.layout.startMs +
+            event.translated.x * state.timelineState.layout.msPerPixel;
+          state.timelineState.contextMenu = {
             type: "trackBody",
             clickMs,
             x: event.x,
             y: event.y,
-            trackId: props.track.id,
+            trackId: state.track.id,
           };
         }
       },
     }),
+    Rect({
```

Comment:
I think this and below Rect are not necessary for *Default*TrackBody. I recommend making SubtitleTrackBody if you want different looks.

Reply:
I divided subtitle track body
with https://github.com/NamseEnt/namseent/pull/18/commits/bb1f6853c663a4bb668e56ab67b94365834a2c1e